### DISCUSSION TITLE
How to reload the player ?

### DISCUSSION THREAD

**Discussion by KM55EN5EI** [PierfrancescoSoffritti/android-youtube-player#1072] [Q&A]
When I click on a button, I want to be able to change the currently playing video and load the player with another video. I cannot use the  nextvideo function as the video I want to load is not from a playlist. So, how can I reload the video player with a new video?

**Comment by PierfrancescoSoffritti** [comment]
You can store the player in a global variable and reuse it to play a new video, using the cue/load video methods. You can see examples in the sample app.
